transaction-impl/src/main/java/com/example/auction/transaction/impl/DeliveryData.java

```diff
@@ -0,0 +1,24 @@
+package com.example.auction.transaction.impl;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import lombok.Value;
+
+@Value
+public class DeliveryData {
```

Comment:
This is the private representation of the `transaction-api`'s `DeliveryInfo`.

We have a terrible mix of naming convetions in that sometimes the private copy is `PXyz`, sometimes it's the exact same name with only a package difference, ...
In general I hate using `Data` and `Info` for anything other than signifying that `data` is master and `info` can be derived from `data`.

I don't think this issue of naming should be solved in this PR but it' be good if we settled and considered creating a convention.